src/components/banner-image/banner-image.js

```diff
@@ -1,31 +1,44 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-export const BannerImage = ({ image: {src, alt, title}, link }) => {
-  return (
-    <div>
-      {link ?
-        <a href={link} target="_blank" rel="noopener noreferrer">
-          <img src={src} alt={alt} title={title}/>
-        </a>
-        :
-        <img src={src} alt={alt} title={title}/>
-      }
-    </div>
+const handleClick = ({ path, shouldOpenLinkOnNewPage }) => {
+  const windowInstance = window.open(
+    path,
+    shouldOpenLinkOnNewPage ? "_blank" : "_self"
   );
+  windowInstance.focus();
 };
 
+export const BannerImage = ({
+  image: { src, alt, title, width, height },
+  link
+}) => (
+  <div>
+    <img
+      src={src}
+      alt={alt}
+      title={title}
+      {...(!!link && { onClick: () => handleClick(link) })}
+      {...(!!width && { width })}
```

Comment:
As far as I know, if the value is `undefined` the prop will not be added. What falsy value do we expect width and height to have so that we need to do this ?

Reply:
Yes, you have right, I just updated this.